src/dotnet/Users.Service/UserContactsBackend.cs

```diff
@@ -133,8 +133,8 @@ public virtual async Task<string> SuggestContactName(string targetUserId, Cancel
 
         if (change.IsCreate(out _))
             await Commander
-                .Call(new IRecentEntriesBackend.UpdateCommand(userContact!.OwnerUserId,
-                        RecentScope.UserContact,
+                .Call(new IRecentEntriesBackend.UpdateCommand(RecentScope.UserContact,
```

Comment:
I'd rather split the line on RecentScope

Reply:
done